lib/perfdata/opentsdbwriter.cpp

```diff
@@ -120,7 +120,7 @@ void OpenTsdbWriter::CheckResultHandler(const Checkable::Ptr& checkable, const C
 	String metric;
 	std::map<String, String> tags;
 
-	String escaped_hostName = EscapeMetric(host->GetName());
+	String escaped_hostName = EscapeTag(host->GetName());
```

Comment:
The fix is that tags are allowed to have "." in them, but using wrong EscapeMetric function was unnecessarily replacing . with _ for all "host" tag. This makes Icinga -> OpenTSDB linking complicated because tags don't match hostnames.